### PR TITLE
Upmcfhir 18/patient fhir3 tests

### PR DESCRIPTION
**Pull request by 5t33**
#### What does this PR do?
UPMC-21
Updates ccdaRoleCode -> patient.contact.relationship.coding to STU3 

UPMC-18
creates initial FHIR3 tests file
#### Related JIRA tickets:
[UPMC-21](https://jira.amida.com/browse/UPMCFHIR-21)
[UPMC-18](https://jira.amida.com/browse/UPMCFHIR-18)

#### How should this be manually tested?
N/A
#### Background/Context
This will drive continuous integration development as we add a post to a hapi server for integration tests. 

How do we feel about these conversions? 

#### New JIRA tickets for clinical review required?

[UPMCFHIR-148](https://jira.amida.com/browse/UPMCFHIR-148)

### CODE REVIEW

File: src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java [hunk 1]
```diff
@@ -725,12 +715,18 @@ public Coding tRoleCode2PatientContactRelationshipCode(String cdaRoleCode) {
...
 			case "stpmth": // stepmother
-				code = "parent"; display = "Parent"; break;
-			case "powatt":
-				code = "agent"; display = "Agent"; break;
+				code="C"; display = "Emergency Contact"; break;
+			case "work": 
+				code = "E"; display = "Employer"; break;
+			case "gt": 
+				code = "BP"; display = "Billing contact person"; break;
+			case "fammemb":
```
Comment (au2) [inline src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java:719]: It looks like BP is deprecated and removed.

File: src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java [hunk 2]
```diff
@@ -690,29 +690,19 @@ public Coding tRoleCode2PatientContactRelationshipCode(String cdaRoleCode) {
...
+		fhirPatientContactRelationshipCode.setSystem("http://hl7.org/fhir/v2/0131");
 		String code = null;
 		String display = null;
 		
 		switch(cdaRoleCode.toLowerCase()) {
 			case "econ": // emergency contact
-				code="emergency"; display = "Emergency"; break;
 			case "ext": // extended family member
-			case "fammemb": // family member
-				code = "family"; display = "Family"; break;
```
Comment (au2) [inline src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java:700]: Can we keep descriptions "// family member" in the code? They are useful. Also family member becomes Next-Of-Kin but father and others become Emergency Contact? Seems inconsistent to me,

**Review by au2 — COMMENTED** [review]
1) Please do not keep commented out import statements in the code.
2) Please add a ticket for future so that test can be moved to a Jolt based one.
3) I do not really like all these switch statements. Static HashMaps are more readable I think. We can discuss later not for this PR.
4) Other reviews are inline.

File: src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java (hunk 1 above)
Comment (5t33) [reply src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java:719]: I'll change it to other.

File: src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java (hunk 2 above)
Comment (5t33) [reply src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java:700]: I guess I considered Next of kin to only be brother/sister? That's exactly why I made this PR. Should all the family related codes be next of kin? What about father in law/mother in law? Does next of kin specifically refer to the closest relative, or are all immediate family members considered "next of kin"? 

I think this is a pretty good example of why we need to get jolt up and running, along with whatever we'll be using to turn it into a flat file so we can get feedback from UPMC.

**Review by au2 — CHANGES_REQUESTED** [review]
You have bunch of unused imports in FHIR3Tests.java; please make sure you have no warnings in Eclipse. Also at some point we need to standardize on the names of these tests; each of us have a different convention. But not this week. Otherwise looks good.

**Comment by 5t33** [comment]
Fixed and removed unused imports. Looks like Eclipse wasn't showing the class name error because it needed a refresh.

**Comment by 5t33** [comment]
Removed unused imports.

**Review by au2 — APPROVED** [review]

**Pull request merged**